cloudinit/config/tests/test_ssh.py

```diff
@@ -1,6 +1,7 @@
 # This file is part of cloud-init. See LICENSE file for license information.
 
 import os.path
+import unittest.mock as umock
```

Comment:
This import isn't needed. The other mock that is imported is essentially the same as this one with an extra convenience "assert" method tacked on.